scss/comments.scss

```diff
@@ -188,3 +73,113 @@ $purple:                #7a43b6 !default;
         margin-top: 10px;
     }
 }
+
+// A published comment
+.comment {
+    clear:both;
+    list-style-type: none;
+    overflow: hidden;
+    padding: 20px 0 10px;
+    position: relative;
+
+    &.author-comment {
+        &:after {
+          content: 'Author';
+          float: right;
+          position: absolute;
+          top: 1.5em;
+          right: 0.5em;
+          font-size: 1em;
+          font-weight: bold;
+      }
+
+      > .comment-text {

```

Comment:
Almost there :D Still looks a bit out here.

If you set your indentation settings to 4/4 and use-tabs maybe it'll simplify the indentation?